### ISSUE TITLE
Hibernation issue

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-gui#2254]
After hibernation on Windows:

```
GNS3 management console.
Running GNS3 version 2.1.0dev4-6e1384c9 on Windows (64-bit) with Python 3.5.0 Qt 5.7.1 and PyQt 5.7.1.
Copyright (c) 2006-2017 GNS3 Technologies.
Use Help -> GNS3 Doctor to detect common issues.

=> QObject::disconnect: Unexpected null parameter
****** Exception detected, traceback information saved in C:\Users\GNS3\AppData\Roaming\GNS3\exceptions.log ******

PLEASE REPORT ON https://www.gns3.com

RecursionError: maximum recursion depth exceeded
```

**Comment by grossmj** [comment]
Also, see https://gns3.com/qa/gns3-stops-working-when-computer

```
This is happening to me using Ubuntu 16.04 when it sleeps or hibernates, I have to literally force quit and open the gns3 project again, hoping that I save all the running configs to startup config. I HAVE FOUND THE PROBLEM AND WORKAROUND SOLUTION. It appears that GNS3 is ties itself the the physical network adapters (eg. my wifi network adapter) even though my topology never uses the cloud for real equipment but only the local IOS Router files, IOU local files and/or the IOSLv2. THE WORKAROUND SOLUTION IS = I turn off "Enable WiFi" and after sleeping or hibernates with black screen and even after short period or long time in that power saving mode, I don't get any crashes. I hope this was included to find another way instead of workaround in the GNS3 2.1, I hate that I cannot have my Internet or network sharing capability open when this bug exists.
```

**Comment by grossmj** [comment]
Happened after hibernation:

```
Error in sys.excepthook:
Traceback (most recent call last):
  File "C:\Python35\lib\site-packages\raven-5.32.0-py3.5.egg\raven\utils\__init__.py", line 185, in new_func
    rv = func(*args, **kwargs)
  File "C:\Python35\lib\site-packages\raven-5.32.0-py3.5.egg\raven\breadcrumbs.py", line 230, in install_logging_hook
    _patch_logger()
  File "C:\Python35\lib\site-packages\raven-5.32.0-py3.5.egg\raven\breadcrumbs.py", line 218, in _patch_logger
    getattr(cls, method_name), level)
  File "C:\Python35\lib\site-packages\raven-5.32.0-py3.5.egg\raven\breadcrumbs.py", line 186, in _wrap_logging_method
    }, logging._srcfile, 'exec'), logging.__dict__, ns)
RecursionError: maximum recursion depth exceeded during compilation
```

**Comment by grossmj** [comment]
This should have been fixed by migrating to Qt 5.9
